### PR TITLE
DEV-6849 update readme

### PR DESCRIPTION
**Pull request by nehemming**
# Pull Request Checklist

- [x] Read the [contributing guidelines](https://github.com/finbourne/lusid-python-tools/blob/master/docs/CONTRIBUTING.md)
- [x] Tests pass

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,60 @@
 # lusid-python-tools
 Python tools for the LUSID
```
Comment (thomasdotred) [inline README.md]: There should be the LUSID banner at the top of this README. Can you try rebasing

File: README.md [hunk 2]
```diff
@@ -1,12 +1,60 @@
...
+The PyPi package for lusid-python-tools can installed globally on your machine using the following command:
+
+```sh
+$ pip3 install lusidtools
+```
+
+or if you are running as a non privileged user you may prefer to install specifically for your user account:
+
+```sh
+$ pip3 install --user lusidtools
```
Comment (rizwansaeed) [inline README.md]: i think we should document as `pip` rather than `pip3`

File: README.md [hunk 3]
```diff
@@ -1,12 +1,60 @@
...
+## Getting Started
+
+You can learn more about LUSID and its benefits by visiting [www.lusid.com](https://www.lusid.com/app/home). 
+
+In order to interact with LUSID, if you have not already done so, go to the [create a LUSID account](https://www.lusid.com/app/signup) page to create your own free trial LUSID domain.
+
+The PyPi package for lusid-python-tools can installed globally on your machine using the following command:
+
+```sh
+$ pip3 install lusidtools
```
Comment (rizwansaeed) [inline README.md]: use `pip`

File: README.md [hunk 4]
```diff
@@ -1,12 +1,60 @@
...
+
+## Reporting Issues
+If you encounter any issues please report these the Github [issues page](https://github.com/finbourne/lusid-python-tools/issues).
+
+## Upgrading
+
+To upgrade lusidtools run one of the commands below 
+
+```sh
+$ pip3 install lusidtools -U
```
Comment (rizwansaeed) [inline README.md]: `pip`

File: README.md [hunk 5]
```diff
@@ -1,12 +1,60 @@
...
+```
+
+or
+
+```sh
+$ pip3 install lusidtools -U --user
```
Comment (rizwansaeed) [inline README.md]: `pip`

File: README.md [hunk 6]
```diff
@@ -1,12 +1,60 @@
...
+
+```sh
+$ pip3 install --user lusidtools
+```
+
+### Testing it works
+
+Once you have an LUSID account you will need to create a developer secrets file.  The steps to do this are covered in this [article](https://support.finbourne.com/getting-started-with-apis-sdks)
+
+Check that the python `bin` folder is on your search path, you may have received a warning for this when you installed the tools above.
```
Comment (rizwansaeed) [inline README.md]: does it also need the lot folder in the path too?

Comment (nehemming) [reply README.md]: I don't have it

File: README.md (hunk 1 above)
Comment (nehemming) [reply README.md]: Done

**Pull request merged**